client/app/src/main/java/com/example/criengine/RequestActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.criengine;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Requested Books Activity
+ * @version  1.0
+ */
+public class RequestActivity extends AppCompatActivity {
+    BorrowerBooksListAdapter borrowerBooksListAdapter;
+    ArrayList<Book> borrowerBooks;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_request);
+
+        // TODO: FIX ME
+        // borrowerBooks = Database.getMyProfile().getbooksBorrowedOrRequested();

```

Comment:
More descriptive TODO? Also, do we know that the return value for this will be a book array? Is this recorded anywhere on our documents? If not, please add!

Reply:
Yep, we added it to the UMLs before db work started